code-challenges.js

```diff
@@ -66,68 +35,26 @@ console.log(multiplyer);
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-//create const variable connected to the function and put the term string within the parenthesis
-const noVowels = (string) => {
-  //create a let variable stating what vowels are
- let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
- //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
- let array = string.split("")
- //create a return statement that will filter through the array and add in .join to join the constanants together
- return array.filter(value => vowels.indexOf(value) === -1).join("")
-}
-//console.log() both variables given
-console.log(noVowels(stringWithVowels1));
-console.log(noVowels(stringWithVowels2))
+
+
 
 
 // --------------------5) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
 // Copy and paste or rewrite the function. Use the test variables provided below.
 
-var stringWithVowels1 = "HeyThereLearnStudent"
-var stringWithVowels2 = "ILoveJavaScript"
+var notAString1 = true
+var notAString2 = 42
+
+
 
-//create const variable connected to the function and put the term string within the parenthesis
-const noVowels = (string) => {
-  //create a let variable stating what vowels are
- let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
- //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
- let array = string.split("")
- //create a return statement that will filter through the array and add in .join to join the constanants together
- return array.filter(value => vowels.indexOf(value) === -1).join("")
```

Comment:
this function needs a bit of refactoring to add some logic to check if the input is actually a string